content/en/getting_started/application/_index.md

```diff
@@ -74,24 +74,23 @@ Dashboards contain [graphs][14] with real-time performance metrics.
 * Share a graph snapshot that appears in the stream.
 * Graphs can be embedded in an iframe. This enables you to give a 3rd party access to a live graph without also giving access to your data or any other information.
 
-## Monitoring
+## Monitors
 
-{{< img src="getting_started/monitor.png" alt="monitor" responsive="true" >}}
+[Monitors][18] provide alerts and notifications based on metric thresholds, integration availability, network endpoints, and more.
 
-[Monitoring][18] gives you the ability to be notified if the aggregate of a specific metric is above or below a certain threshold—across your entire infrastructure, or per machine (average, max, min, or sum).
+* Use any metric reporting to Datadog
+* Set up multi-alerts (by device, host, etc.)
+* Alert notification messages have `@` capabilities
+* Schedule downtimes for system shutdowns, off-line maintenance, etc.
```

Comment:
`Schedule downtimes to suppress notifications for system shutdowns, off-line maintenance, etc.`